src/handler/http/models/alerts/destinations.rs

```diff
@@ -0,0 +1,143 @@
+// Copyright 2024 OpenObserve Inc.
```

Comment:
Struct definitions here are all copy-pasted from `config::meta` since, prior to this PR, the JSON de/serialization logic in `config::meta` defines the HTTP API contracts.